ferrocene/ci/scripts/cache.py

```diff
@@ -1,90 +1,21 @@
-#!/usr/bin/env python3
+#!/usr/bin/env -S uv run
```

Comment:
A particular HN commentator suggests shebang approach fails if you are ever offline (such as in a sandbox), perhaps this is a concern for us? https://news.ycombinator.com/item?id=42201103

Reply:
At least for these scripts no.